### PR TITLE
Styling, attempt #3

### PR DESCRIPTION
**Pull request by Siianko**
screenshot 2018-08-29 12 09 59

https://www.dropbox.com/s/9dg3rmauhroglks/Screenshot%202018-08-29%2012.12.26.png?dl=0

https://www.dropbox.com/s/2sessivoeij3m5y/Screenshot%202018-08-29%2012.12.50.png?dl=0

https://www.dropbox.com/s/wx3zpwuhjiy8701/Screenshot%202018-08-29%2012.13.53.png?dl=0

https://www.dropbox.com/s/d4an6d7emi0175d/Screenshot%202018-08-29%2012.14.05.png?dl=0

### CODE REVIEW

**Comment by magnus-thor** [comment]
Add some screenshots of the styling you have done

File: app/views/devise/registrations/edit.html.haml [hunk 1]
```diff
@@ -1,15 +1,15 @@
-%h2
+%h2.m-4.text-black.text-center
   Edit #{resource_name.to_s.humanize}
 = form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f|
   = devise_error_messages!
-  .field
+  .field.block.text-black.m-2.font-bold.md:text-center.mb-1.md:mb-0.pr-4
```
Comment (magnus-thor) [inline app/views/devise/registrations/edit.html.haml]: I want you to move these classes to the css file. In the html file you leave the `.field` and move the rest to the application.css, it should look like this
```
.field {
  @apply block text-black m-2 font-bold pr-4 mb-1;
}

@screen md {
  .field {
    @apply text-center mb-0;
  }
} 
```

File: app/views/articles/new.html.haml [hunk 2]
```diff
@@ -1,16 +1,24 @@
...
+  .block.text-black.m-2.font-bold.md:text-center.mb-1.md:mb-0.pr-4
+    = form.file_field :image
   %br/
-  = form.submit t('create_article')
+  .block.text-black.m-2.font-bold.md:text-center.mb-1.md:mb-0.pr-4
+    = form.submit t('create_article')
 
-= link_to t('back'), articles_path
\ No newline at end of file
+.block.text-black.m-2.font-bold.md:text-center.mb-1.md:mb-0.pr-4
```
Comment (magnus-thor) [inline app/views/articles/new.html.haml]: all of these can be moved to the css file, look at next comment for instructions on how to move it.

**Review by magnus-thor — CHANGES_REQUESTED** [review]

File: app/views/shared/_links_navbar.html.haml [hunk 3]
```diff
@@ -1,12 +1,27 @@
...
+    .notice
+        = flash[:notice]
+    .alert
+        = flash[:error]
+    .container.md:flex.md:items-center.md:justify-between.mx-auto.px-5
+        %h1.mb-4.md:mb-0.mx-auto.text-2xl= link_to 'D-pressed', root_path 
+        %ul.md:flex.md:items-center.list-reset.text-xl
+            %li.border-t.md:border-0.md:ml-4
+                - unless current_user
+                    %a.block.md:inline.no-underline.py-4.px-4.md:py-0.text-white.hover:text-grey-lighter
```
Comment (magnus-thor) [inline app/views/shared/_links_navbar.html.haml]: this should also be moved to the css file.

**Review by magnus-thor — CHANGES_REQUESTED** [review]

File: app/views/shared/_links_navbar.html.haml (hunk 3 above)
Comment (magnus-thor) [reply app/views/shared/_links_navbar.html.haml]: replace the line with a class name ( I used `.links` ) and then in the css file
```
.links {
  @apply block no-underline py-4 px-4 text-white;
}

.links:hover {
  @apply text-grey-lighter;
}

@screen md {
  .field {
    @apply text-center mb-0;
  }

  .links {
    @apply inline py-0;
  }
}
```

Comment (Andycutter) [reply app/views/shared/_links_navbar.html.haml]: I will remove this Tailwind code completely and deal with the styling of the links in the application.css on its own.

**Review by magnus-thor — APPROVED** [review]
looks good, Good job.

**Pull request merged**